collection_requirements.rst

```diff
@@ -60,7 +62,9 @@ Documentation
 All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
 
 * Use the FQCN for ``M(...)`` and ``- module:`` references of ``seealso`` subsections. See `Linking within module documentation <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
-* Use collection version numbers for `version_added` (otherwise `version_added_collection` must be provided, but this is discouraged)
+* Use collection version numbers for ``version_added``, and not Ansible version numbers or other unrelated version numbers.
```

Comment:
should the sentence be rephrased
> use field `version_added` to define the version the module appeared in the collection. Don't define Ansible version numbers or other unrelated version numbers.

Reply:
I adjusted the sentence. Can you take another look?